arviz/plots/backends/bokeh/forestplot.py

```diff
@@ -607,40 +669,42 @@ def iterator(self):
                 else:
                     row_label = label
                 for values in value_list:
-                    yield y, row_label, label, selection_list[idx], values, self.model_color[
-                        model_name
-                    ]
+                    yield y, row_label, model_name, label, selection_list[
```

Comment:
This is for the code above, not for here, but you need to update the label generation to use the labeller: https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/matplotlib/forestplot.py#L555

Reply:
By this, you just mean that I need to generate the row labels as done In matplotib?

Reply:
Both row labels and model labels used in legend should be processed by the labeller, you'll see that in matplotlib there are two labeller methods called